src/main/java/com/commercetools/paymenttoorderprocessor/timestamp/TimeStampManagerImpl.java

```diff
@@ -90,20 +92,27 @@ private void queryTimeStamp() {
         if (results.isEmpty()) {
             LOG.warn("No Timestamp for last processed message for was found at commercetools platform. This should only happen on the first run.");
         } else {
-            lastTimestamp = Optional.of(results.get(0));
-            LOG.info("Got Timestamp from commercetools platform: [{}] ",
-                    lastTimestamp.map(CustomObject::getValue).map(TimeStamp::getLastTimeStamp).map(Object::toString).orElse("null"));
+            lastTimestamp = results.get(0);
+            LOG.info("Got Timestamp from commercetools platform: [{}] ", timestampToString(lastTimestamp));
```

Comment:
i'd say this log exactly fits "info" definition: it is an information about the application execution and show how the app going on. If something is crashed in the future we at least have some checkpoints of the application execution. Also it will be useful to compare how the previous application finished and a new one is started.